### PR TITLE
feat(issue-cmd): added new GH issue command

### LINKED ISSUE #932: ignite doctor --bug should work
In [the docs](https://github.com/infinitered/ignite/blob/1c677e1e53b6de27a4c9386e5aa2c6a980fe8071/docs/quick-start/ignite-commands.md#doctor), we mention `ignite doctor --bug` which ostensibly should open a browser with an Ignite issue ready to go and the results of `ignite doctor` pre-filled.

I'm not sure where this idea came from, but it's a good one. We should implement it.

### PR DESCRIPTION
**Pull request by frankcalise**
## Describe your PR
- Closes #932 
- Adds new `npx ignite-cli issue "my bug title here"` command
- Opens browser window to `https://github.com/infinitered/ignite/issue/new` with title prefilled and prefilled issue body with `npx ignite doctor` output for issue reporter to easily paste
- Fixes title for user if they forget to double quote the title string

Tested
- [x] MacOS
- [x] Windows

image

image

### CODE REVIEW

**Comment by frankcalise** [comment]
Thoughts @joshuayoes @jamonholmgren ?

**Comment by joshuayoes** [comment]
I love this! I know you mentioned the GitHub CLI but I like this approach. I think allowing people to quickly open up an issue and then edit it in the browser is the right approach here.

**Comment by frankcalise** [comment]
@joshuayoes actually I think I can get it to paste the body correctly, was passing the url params incorrectly. however, do you know of any way to capture the output from `system.run` without colors? I think the colored output from `npx ignite-doctor` is messing things up here

image

**Comment by joshuayoes** [comment]
You should be able to use the strip-ansi package, we use it for the test output: https://github.com/infinitered/ignite/blob/master/test/_test-helpers.ts#L2

**Comment by frankcalise** [comment]
@joshuayoes wow awesome thanks! worked perfectly

image

**Comment by frankcalise** [comment]
I think it could be cool to search the existing issues for the passed in title to output a few of the links to the results in the terminal for them to look up prior to submitting, but this implementation is probably a good first step to start with.

**Review by joshuayoes — APPROVED** [review]

File: src/commands/issue.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  startSpinner,
+  stopSpinner,
+  prettyPrompt,
+  prefix,
+} from "../tools/pretty"
+const stripANSI = require("strip-ansi")
+
+const isMac = process.platform === "darwin"
+
+const START_CMD = isMac ? "open" : "start"
```
Comment (jamonholmgren) [inline src/commands/issue.ts:19]: Won't work on Linux, but we'll let them fix with their own command.

Comment (jamonholmgren) [reply src/commands/issue.ts:19]: And of course it's ridiculous over there: https://stackoverflow.com/questions/5116473/linux-command-to-open-url-in-default-browser

Comment (jamonholmgren) [reply src/commands/issue.ts:19]: We could use `open` -- https://github.com/sindresorhus/open

Or this command, which won't work with all distros, but get us mostly there: https://stackoverflow.com/a/49013356/204044

But for this PR, let's not worry about it.

File: README.md [hunk 2]
```diff
@@ -92,6 +92,8 @@ Note:
 
 If you run into problems, first search the issues and discussions in this repository. If you don't find anything, you can come talk to our friendly and active developers in the Infinite Red Community Slack ([community.infinite.red](http://community.infinite.red)).
 
+If you still need help after reaching out to the proper channels, feel free to open a new GitHub issue via `npx ignite-cli issue "Unable to Ignite new app"` (as an example). This will help start writing your issue with the correct diagnostic information included.
```
Comment (jamonholmgren) [inline README.md:95]: We really need an `ignite-cli` doc, documenting all commands. Not in this PR though, filed an issue #2291.

**Review by jamonholmgren — APPROVED** [review]

File: src/commands/issue.ts (hunk 1 above)
Comment (frankcalise) [reply src/commands/issue.ts:19]: Ah was hoping there was some cross platform *nix usage with `open` but I guess not

**Pull request merged**